backend-go/go.mod

```diff
@@ -1,24 +1,13 @@
 module github.com/wuespace/telestion/backend-go
 
-go 1.21.4
+go 1.23
 
-require (
-	github.com/cucumber/godog v0.13.0
-	github.com/mitchellh/mapstructure v1.5.0
-	github.com/nats-io/nats.go v1.31.0
-)
+require github.com/nats-io/nats.go v1.37.0
 
 require (
-	github.com/cucumber/gherkin/go/v26 v26.2.0 // indirect
-	github.com/cucumber/messages/go/v21 v21.0.1 // indirect
-	github.com/gofrs/uuid v4.4.0+incompatible // indirect
-	github.com/hashicorp/go-immutable-radix v1.3.1 // indirect
-	github.com/hashicorp/go-memdb v1.3.4 // indirect
-	github.com/hashicorp/golang-lru v1.0.2 // indirect
-	github.com/klauspost/compress v1.17.4 // indirect
+	github.com/klauspost/compress v1.17.2 // indirect
```

Comment:
Any reason for the downgrade? Current version would be v1.17.11?

Reply:
I really have no idea. I _think_ I installed the package through the CLI without specifying a version and that was the result. I could be wrong. Will look into it.

Reply:
Updated.